generators/language-python-flask/index.js

```diff
@@ -104,11 +103,12 @@ module.exports = class extends Generator {
 
 	_addDependencies(serviceDepdendenciesString){
 		let requirementsTxtPath = this.destinationPath(PATH_REQUIREMENTS_TXT);
-		let pipfileUserPath = this.destinationPath(PATH_PIPFILE);
+		let pipfileUserPath = this.destinationPath('Pipfile');
```

Comment:
I still don't understand why `PATH_PIPFILE` was removed here. Wouldn't it make more sense to just change `const PATH_PIPFILE = "Pipfile.txt";` to `const PATH_PIPFILE = "Pipfile";`?

Reply:
Oh yeah sorry forgot to fix this